main_file.py

```diff
@@ -1,30 +1,23 @@
 from shared_vae_class import shared_vae_class
 from model_objects import model_parameters
-
-
 import keras
-from keras.datasets import mnist
-
 import numpy as np
+import ICVL
+import MNIST
+import os
 
-# Loading the MNIST Data
-(x_train, _), (_, _) = mnist.load_data()
-
-# Formating the MNIST Data
-x_train = x_train.astype('float32') / 255.
-x_train = x_train.reshape((len(x_train), np.prod(x_train.shape[1:])))
-
-# Making Copies of the Data to creat Inverses
-import copy
+# MNIST 28x28
+# ICVL 60x80
 
-a_train = copy.copy(x_train)
-for i in range(len(a_train)):
-    a_train[i] = 1 - a_train[i]
+dataSetInfo = ICVL.dataInfo()
```

Comment:
I'm not sure if I can come up with a benefit which is exclusive with giving them specific names. It made more sense to me because they're implementations of the abstract function, so by using the same name they can interface really easily with only having to change which file they're called from. I figured that this would allow for a dictionary down the road which would mean that the user could input a dataset when they run the program, and not have to change anything in the files at all to switch between datasets. This, however, is basically just as easy to do with specific names. I originally gave them specific names but changed it. Basically, it just made the most sense to me in my head to emphasize that the functions interface the same way.